code/modules/admin/verbs/purrbation.dm

```diff
@@ -0,0 +1,45 @@
+/proc/mass_purrbation()
+	for(var/M in GLOB.mob_list)
```

Comment:
use descript variable names for all these please
also i think we have better global lists for that, like living mobs, or players.